src/sentry/static/sentry/app/views/alerts/rules/index.tsx

```diff
@@ -82,6 +100,18 @@ class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state
     );
   }
 
+  handleChangeFilter = (activeFilters: Set<string>) => {
+    const {router, location} = this.props;
+    this.setState({filteredTeams: activeFilters});
+    router.push({
```

Comment:
Why `setState` if we are going to push to the same route and get the state from route params anyways?

Reply:
good catch will change